ansible/README.md

```diff
@@ -0,0 +1,36 @@
+<!--
+ Copyright (c) 2017 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# Ansible swan deployment playbook
+
+## Introduction
+To deploy your own swan experiments to remote nodes we provide a environment preparation scripts both for your host and remote machines. Note that currently host preparation script uses `apt` package manager so `Debian` or `Ubuntu` would be a great choice. Remote installation scripts(ansible) on the other hand, are written for `CentOS7`. 
+
+## Usage
+First, to prepare your own machine for development and build Swan locally run following:
+```bash
+git clone https://github.com/intelsdi-x/swan.git
```

Comment:
@Dual2Core use `go get` instead. Cloning golang source code to location outside of `GOPATH` is not a good idea.

Reply:
I get your idea, but `prepare_host_env.sh` assumes that there is no Golang installed, so I shouldn't tell to use it to download files to install Golang. That would be strange. So I can leave it like it is in the tutorial or remove whole Go installation part and leave a note in requirements section to have go installed before running the script. However, I can also add `rm -rf swan` and then Swan repo will exist only at Gopath location.

Reply:
If we download the project source just in order to get hold of Ansible manifests and shell scripts then cloning is good solution.

Reply:
@Dual2Core I do like your idea about replacing `git clone` with downloading single shell script. Implement it, please. 👍